builtin/providers/aws/resource_aws_route53_zone.go

```diff
@@ -171,6 +171,10 @@ func resourceAwsRoute53ZoneRead(d *schema.ResourceData, meta interface{}) error
 		d.Set("delegation_set_id", cleanDelegationSetId(*zone.DelegationSet.Id))
 	}
 
+	if zone.HostedZone != nil && zone.HostedZone.Config != nil && zone.HostedZone.Config.Comment != nil {
```

Comment:
I don't think the third condition is necessary since `d.Set` should be able to handle `nil`s, but it does no harm. :wink: